install-timelord.sh

```diff
@@ -102,8 +109,8 @@ else
 	elif [ -e venv/bin/python ] && test "$RHEL_BASED"; then
 		echo "Installing chiavdf dependencies on RedHat/CentOS/Fedora"
 		# Install remaining needed development tools - assumes venv and prior run of install.sh
-		echo "yum install gcc gcc-c++ gmp-devel $PYTHON_DEV_DEPENDENCY libtool make autoconf automake openssl-devel libevent-devel boost-devel python3 cmake -y"
-		sudo yum install gcc gcc-c++ gmp-devel "$PYTHON_DEV_DEPENDENCY" libtool make autoconf automake openssl-devel libevent-devel boost-devel python3 cmake -y
+		echo "yum install gcc gcc-c++ gmp-devel $PYTHON_DEV_DEPENDENCY libtool make autoconf automake openssl-devel libevent-devel boost-devel python3 cmake -y" | tr -s ' '
+		sudo yum install gcc gcc-c++ gmp-devel "$PYTHON_DEV_DEPENDENCY" libtool make autoconf automake openssl-devel libevent-devel boost-devel python3 cmake -y | tr -s ' '
```

Comment:
I don't have a centos system to test with, but I ran this in a rockylinux container, and I don't think this has the desired effect. It would seem that the `sudo yum` command gets ran as-is before the output of which gets piped to `tr`

Reply:
Perhaps something like this would do it?

```bash
yumcmd=$(echo "sudo yum install gcc gcc-c++ gmp-devel $PYTHON_DEV_DEPENDENCY libtool make autoconf automake openssl-devel libevent-devel boost-devel python3 cmake -y" | tr -s ' ')
echo "$yumcmd"
${yumcmd}
```

What do you think @LeroyINC ?

Reply:
good catch... and that makes sense.

how about this to make it more simple

if [ "$(uname)" = "Linux" ] && type apt-get; then
	UBUNTU_DEBIAN=true
	echo "Found Ubuntu/Debian."
elif [ "$(uname)" = "Linux" ] && type dnf || yum; then
	RHEL_BASED=true
	echo "Found RedHat."
	
	if [ "$INSTALL_PYTHON_DEV" ]; then
		yumcmd="sudo yum install "$PYTHON_VERSION"-devel gcc gcc-c++ gmp-devel libtool make autoconf automake openssl-devel libevent-devel boost-devel python3 cmake -y"
	else
		yumcmd="sudo yum install gcc gcc-c++ gmp-devel libtool make autoconf automake openssl-devel libevent-devel boost-devel python3 cmake -y"
	fi
	
elif [ "$(uname)" = "Darwin" ]; then
	MACOS=true
	echo "Found MacOS."
fi


then lower down... 

echo "$yumcmd"
${yumcmd}

Reply:
updated the code... check it out and let me know

Reply:
Thanks for the update! Left one more comment, and I'll resolve this one.